projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/service/KubernetesServiceTest.java

```diff
@@ -341,24 +339,6 @@ public void testReadJobDeploymentStatuses() {
     Assertions.assertEquals(mergedTestLists, resultStatuses);
   }
 
```

Comment:
This test is testing nothing anymore. It is mocking a response from a method and then just asserting the response is what we except